src/output.jl

```diff
@@ -51,7 +59,11 @@ plots the temperature of a landscape for a given timestep t
 """
 function image_temperature(SD::Simulation_Data, t::Int)
```

Comment:
here too. Looks like the general heatmap ones could be flipped 
![grafik](https://github.com/janablechschmidt/MetaRange.jl/assets/77731816/039567c1-8594-4359-9e5d-406f704c831d)

Reply:
should be fixed now!